app/Voucher.php

```diff
@@ -41,14 +42,13 @@ public function sponsor()
         return $this->belongsTo(Sponsor::class);
     }
 
-    public function redeemer()
-    {
-        return $this->belongsTo(User::class);
-    }
-
     public function creditor()
     {
         return $this->belongsTo(Trader::class);
```

Comment:
Why not stick to one or other of App\Trader vs Trader::class vs use App\Trader; Trader